official/resnet/resnet_run_loop.py

```diff
@@ -645,6 +683,15 @@ def define_resnet_flags(resnet_size_choices=None):
           'the expense of image resize/cropping being done as part of model '
           'inference. Note, this flag only applies to ImageNet and cannot '
           'be used for CIFAR.'))
+  flags.DEFINE_string(
+      name='worker_hosts', default=None,
+      help=flags_core.help_wrap(
+          'Comma-separated list of worker ip:port pairs for running '
+          'multi-worker models with DistributionStrategy.'))
+  flags.DEFINE_integer(
```

Comment:
Are you planning to update the README in this directory to guide users how to use multi worker training? (Can be a separate PR but would be good to update)

Reply:
Thanks for that suggestion, will do